stdlib/influxdata/influxdb/from.go

```diff
@@ -4,18 +4,46 @@
 package influxdb
 
 import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
 	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/execute"
 	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/memory"
 	"github.com/influxdata/flux/plan"
 	"github.com/influxdata/flux/runtime"
 	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/stdlib/universe"
+	"github.com/influxdata/flux/values"
+)
+
+const (
+	FromKind       = "from"
+	FromRemoteKind = "influxdata/influxdb.fromRemote"
 )
 
-const FromKind = "from"
+// NameOrID signifies the name of an organization/bucket
+// or an ID for an organization/bucket.
+type NameOrID struct {
+	ID   string
+	Name string
+}
 
```

Comment:
I just wanna understand, so in this iteration of from, we're able to accept an organization name, a bucket name, an organization ID or a bucket ID? Confirming because that's different from what I previously understood.

Reply:
We are able to accept a bucket name or a bucket id and we are also able to accept an organization name or organization id. So you can do `from(org, bucket)`, `from(orgID, bucket)`, `from(org, bucketID)`, or `from(orgID, bucketID)`. But you can't do `from(org, orgID)` or `from(bucket, bucketID)` and for flux itself, you are going to need to specify both the bucket and org. For the influx cli, you won't need to specify the org.